### ISSUE TITLE
[BUG] Error on order table by column in MSSQL/Sql Server

### ISSUE THREAD

**Issue by McGuten** [cweijan/vscode-mysql#757]
When i order a table the SQL generated has a error:

```SQL
SELECT TOP 500 * FROM <table name>; ORDER BY "<column>" desc
```
The ";" its in wrong position.

- Operating system: WSL " over ubuntu
- Database type and version: MSSQL 2019 Express

**Comment by cweijan** [comment]
Thanks for the feedback, I will fix it in the next version.

**Comment by cweijan** [comment]
Fix in version 6.1.6.
